kafka-ui-react-app/src/components/Topics/List/List.tsx

```diff
@@ -72,7 +88,20 @@ const List: React.FC<Props> = ({
               <label htmlFor="switchRoundedDefault">Show Internal Topics</label>
             </div>
           </div>
-          <div className="level-item level-right">
+          <div className="column">
+            <p className="control has-icons-left">
+              <input
+                className="input"
+                type="text"
+                placeholder="Search by Topic Name"
+                onChange={handleSearch}
+              />
+              <span className="icon is-small is-left">
+                <i className="fas fa-search" />
+              </span>
+            </p>
```

Comment:
I would suggest to make search reusable and move it to the own common/search component